ext/openssl/openssl.stub.php

```diff
@@ -22,3 +22,49 @@ function openssl_x509_read($cert) {}
 
 /** @return false|void */
 function openssl_x509_free($x509) {}
+
+function openssl_pkcs12_export_to_file($x509cert, string $filename, $priv_key, string $pass, array $args = UNKNOWN): bool {}
+
+function openssl_pkcs12_export($x509 , &$out, $priv_key, string $pass, array $args = UNKNOWN): bool {}
+
+function openssl_pkcs12_read(string $pkcs12, &$certs, string $pass): bool {}
+
+function openssl_csr_export_to_file($csr, string $outfilename, bool $notext = true): bool {}
+
+function openssl_csr_export($csr, &$out, bool $notext = true): bool {}
+
+/** @return resource|false */
+function openssl_csr_sign($csr, $cacert, $priv_key, int $days, array $config_args = UNKNOWN, int $serial = 0) {}
```

Comment:
A `!` in the `zend_parse_parameters()` format signals that the previous parameter is nullable, so:
```suggestion
/**
 * @param resource|string $csr
 * @param resource|string $cacert
 * @param resource|string|array $priv_key
 * @return resource|false
 */
function openssl_csr_sign($csr, $cacert = null, $priv_key, int $days, ?array $config_args = null, int $serial = 0) {}
```